### PR TITLE
Add license footer to all documents under /curriculum

### PR DESCRIPTION
**Pull request by breville**
English only for now, like so:

screenshot 2016-05-10 11 49 38

### CODE REVIEW

**Comment by breville** [comment]
fyi @Hamms

**Comment by Hamms** [comment]
Although I never saw the actual spec, this looks good to me. Do we intend to internationalize this eventually?

**Comment by breville** [comment]
Eventually, but Tanya says we don't need it to be loc'ed yet.

**Comment by tanyaparker** [comment]
Right, since we don't localize these lesson plans yet it's OK to keep it English. We'll localize it as a whole...eventually.

File: pegasus/sites/code.org/views/page_curriculum.haml [hunk 1]
```diff
@@ -18,3 +18,17 @@
   %body
     .container_nonresponsive{:style=>'width:100%'}
       = body
+
+    .container_nonresponsive{:style=>'width: 100%'}
```
Comment (tanyaparker) [inline pegasus/sites/code.org/views/page_curriculum.haml:22]: So should we be gravitating towards `:style=>` or `style:` in haml? I don't care which one, but think the same file should stick to one.

**Comment by breville** [comment]
Oh good point.  I think we should gravitate towards style:

File: pegasus/sites/code.org/views/page_curriculum.haml [hunk 2]
```diff
@@ -18,3 +18,17 @@
...
+      .license_footer{style: "font-size: 13px"}
+        .license-image{style: "float: left; margin-right: 15px; margin-top: 5px;"}
+          %a{href: "http://creativecommons.org/", target: "_blank", style: "text-decoration: none"}
+            %img{src: "/images/curriculum/cc-license-88x31.png"}
+        .license-text{style: "float: left"}
+          %a{href: "http://creativecommons.org/", target: "_blank", style: "text-decoration: none"}>
+            This curriculum is available under a Creative Commons License (CC BY-NC-SA 4.0)
+          %br/
+          If you are interested in licensing
+          %a{href: "/"} Code.org
```
Comment (tanyaparker) [inline pegasus/sites/code.org/views/page_curriculum.haml:32]: How do links in lesson plans work? Do we want to use CDO.code_org_url here?

**Comment by breville** [comment]
I thought that we were fine to use relative links since we know that these pages are hosted on code.org.

**Comment by breville** [comment]
Or are you saying these get embedded on studio.code.org somehow?

File: pegasus/sites/code.org/views/page_curriculum.haml [hunk 3]
```diff
@@ -18,3 +18,17 @@
   %body
     .container_nonresponsive{:style=>'width:100%'}
       = body
+
+    .container_nonresponsive{:style=>'width: 100%'}
+      .license_footer{style: "font-size: 13px"}
+        .license-image{style: "float: left; margin-right: 15px; margin-top: 5px;"}
```
Comment (tanyaparker) [inline pegasus/sites/code.org/views/page_curriculum.haml:24]: Could these in-line styles be added to curriculum.css?

**Comment by tanyaparker** [comment]
Well if we link to it from teacher dashboard (when eventually the dashboard is on Code Studio), I guess it'll still point to where it lives on pegasus. So probably not necessary.

**Pull request merged**